### ISSUE TITLE
Warn about unused As imports

### ISSUE THREAD

**Issue by ndmitchell** [ndmitchell/hlint#330]
If someone does an `import Foo as Bar`, and the qualified name `Bar` is never used, then warn.

**Comment by mitchellwrosen** [comment]
Just to clarify, this is not meant to catch code like the following, correct?

```haskell
import Foo as Bar
import Woo as Bar
x = Bar.y -- comes from Foo, not Woo
```

**Comment by ndmitchell** [comment]
That seems impossible to detect with the information HLint knows.
